contrib/drivers/huawei/dorado/replication.go

```diff
@@ -269,7 +269,7 @@ func (r *ReplicaPairMgr) CreateRemoteLun(localLun *Lun) (*Lun, error) {
 	if err != nil {
 		return nil, err
 	}
-	rmtLun, err := r.remoteClient.CreateVolume(localLun.Name, sector, localLun.Description)
+	rmtLun, err := r.remoteClient.CreateVolume(localLun.Name, sector, localLun.Description, "0")
```

Comment:
Will be remote lun be created on pool 0?  What about if pool 0 doesn't exist?

Reply:
These code is not used and  will be removed next version, modified it here is just for building error.

Reply:
This code is design for creating remote lun when user not provide the remote lun id. But in current version, user should create local lun and remote lun manually, then create the pair relationship. So it  is not used.